### ISSUE TITLE
[Supporting the platform - Logic Apps - API Connections]: Single Pointer options alternative is not provided for adjusting the width of table columns present in the 'API Connections' page.

### ISSUE THREAD

**Issue by takyyon** [Azure/LogicAppsUX#6113]
### Describe the issue 

User Experience:

 Functionality that uses a dragging movement should have another single pointer mode of operation without the need for the dexterity required to drag elements. Users who struggle with performing dragging movements can easily operate an interface with a pointer interface.

 Note: User credentials should NOT be included in the bug.

Repro Steps:

- Open [https://ms.portal.azure.com/](<https://ms.portal.azure.com/>) and login via valid credentials. 
- search 'Integration Service Environment' and Select any already created Integration Service Environment 
- Select 'API Connections' under Setting in left navigation pane. Navigate all the controls on the blade. 
- Verify whether the Single Pointer options alternative is provided for adjusting the width of table columns or not. 

Actual Result:

 Single Pointer options alternative is not provided for adjusting the width of table columns present in the 'API Connections' page.

Expected Result:

 Single Pointer options alternative should be provided for adjusting the width of table columns present in the 'API Connections' page.

 Example: There can be options provided in the dropdown for column headers to input the position number where the specific column needs to be moved and enter the desired width in pixels for that column.

Note:

 This same issue exists on the 'Custom Connectors' page.

 The same issue exists under 'Run history' blade. (Overview>Run history tab>Select any>Run history blade)

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

*No response*

### Browser 

Chrome, Edge

### Additional context 

*No response*

**Comment by hartra344** [comment]
ISE is deprecated
